channels/smartcard/client/smartcard_pack.c

```diff
@@ -2813,7 +2813,7 @@ LONG smartcard_pack_get_attrib_return(SMARTCARD_DEVICE* smartcard, wStream* s,
 {
 	LONG status;
 	DWORD cbAttrLen;
-	DWORD index = 0;
+	UINT32 index = 0;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW